include/lsst/meas/extensions/trailedSources/VeresModel.h

```diff
@@ -0,0 +1,100 @@
+// -*- lsst-c++ -*-
+/*
+ * This file is part of meas_extensions_trailedSources.
+ *
+ * Developed for the LSST Data Management System.
+ * This product includes software developed by the LSST Project
+ * (https://www.lsst.org).
+ * See the COPYRIGHT file at the top-level directory of this distribution
+ * for details of code ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#ifndef LSST_MEAS_EXTENSIONS_TRAILEDSOURCES_VERES_H
+#define LSST_MEAS_EXTENSIONS_TRAILEDSOURCES_VERES_H
+
+#include "lsst/pex/config.h"
+#include "lsst/afw/image.h"
+#include "lsst/afw/geom.h"
+#include "lsst/geom.h"
+#include "ndarray.h"
```

Comment:
Sorting [by type and dependency](https://developer.lsst.io/cpp/style.html#include-statements-should-be-sorted-and-grouped) is preferred:
```suggestion
#include "ndarray.h"

#include "lsst/pex/config.h"
#include "lsst/geom.h"
#include "lsst/afw/geom.h"
#include "lsst/afw/image.h"
```